metrics/metrics-core/src/main/java/io/servicecomb/metrics/core/metric/InstanceMetric.java

```diff
@@ -19,7 +19,8 @@
 
 public class InstanceMetric extends ModelMetric {
   public InstanceMetric(long waitInQueue, TimerMetric lifeTimeInQueue,
-      TimerMetric executionTime, TimerMetric consumerLatency, TimerMetric producerLatency) {
```

Comment:
We should know if the invocation is consumer or producer to avoid storing other two metrics here.

Reply:
Done,Separate into tow metric named "ConsumerInvocationMetric" and "ProducerInvocationMetric"